custom_components/reflex_ag_grid/ag_grid.py

```diff
@@ -104,10 +104,11 @@ def _api(self) -> rx.Var:
         return f"refs['{self.ref}']?.current?.api"
 
     def __getattr__(self, name: str) -> Callable[[Any], rx.event.EventSpec]:
-        def _call_api(*args):
+        def _call_api(*args, **kwargs):
```

Comment:
i think a docstring mentioning that `args` goes to the function and `kwargs` goes to the `rx.call_script` would be helpful for future maintainers